test/pylib/cpp/boost/boost_facade.py

```diff
@@ -123,25 +123,17 @@ def read_file(name: Path) -> str:
         if not test_passed:
             allure.attach(stdout_file_path.read_bytes(), name='output', attachment_type=allure.attachment_type.TEXT)
             msg = (
-                'working_dir: {working_dir}\n'
-                'Internal Error: calling {executable} '
-                'for test {test_id} failed (return_code={return_code}):\n'
-                'output file:{stdout}\n'
-                'log:{log}\n'
-                'command to repeat:{command}'
+                f'working_dir: {os.getcwd()}\n'
+                f'Internal Error: calling {executable} '
+                f'for test {test_name} failed ({return_code=}):\n'
+                f'output file:{stdout_file_path.absolute()}\n'
+                f'log:{log}\n'
+                f'command to repeat:{"".join(args)}\n'
```

Comment:
It used to be `' '.join(args)`, but now it is `"".join(args)` - notice the lack of space in the string that we are joining by here. Was this intended?